cogs/utils/constants.py

```diff
@@ -228,6 +228,713 @@
     default = safe_load(file)
 
 
+talk_games = {
```

Comment:
Unacceptable, use a `.txt` file in `assets` or direcrtly in `games/` (after #62) this is way too big for `constants.py`